examples/tutorials/tacotron2_pipeline_tutorial.py

```diff
@@ -256,7 +203,7 @@ def plot():
 # WaveRNN model from the same bundle.
 #
 
-bundle = torchaudio.pipelines.TACOTRON2_WAVERNN_PHONE_LJSPEECH
+bundle = torchaudio.pipelines.TACOTRON2_WAVERNN_CHAR_LJSPEECH
```

Comment:
Something is off in this tutorial as a result of the change. The section above about spectrogram generation is still using `TACOTRON2_WAVERNN_PHONE_LJSPEECH`. And just above, the tutorial says

> Continuing from the previous section, we can instantiate the matching WaveRNN model from the same bundle


but the bundle has changed from `TACOTRON2_WAVERNN_PHONE_LJSPEECH` to `TACOTRON2_WAVERNN_CHAR_LJSPEECH`.

Can you please advise on how to resolve the mismatch?

Reply:
We should never be using `TACOTRON2_WAVERNN_PHONE_LJSPEECH`, as this will require the Deep-Phonemizer library. I must have missed some references to it if you're still seeing some.

Reply:
I should have clarified at the start of the PR: the way I propose to deal with the Deep-Phonemizer dependency here is to swap out the `TACOTRON2_WAVERNN_CHAR_LJSPEECH` pipeline (which us a character-level model) for the TACOTRON2_WAVERNN_PHONE_LJSPEECH pipeline (which is a phoneme level model and requires the dependency). Most of the tutorial can continue unchanged (except for the section comparing character-level to phoneme level model results).